mantidproject/mantid
# Fix IDA preview plotting

Fixes [#11167](http://trac.mantidproject.org/mantid/ticket/11167).

To test:
- Download the files from the ticket
- Open MSDFit
  - Load `osi92762_graphite002_elwin_eq2.nxs`
  - Click Fit Single Spectrum, should see a fit curve that will disappear when you change Plot Spectrum
  - Click Run, sill have a fit curve for all spectra that persists until another fit is done
- Open FuryFit
  - Load `irs26176_graphite002_iqt.nxs`
  - Move EndX such that is misses out the garbage at the end of the spectrum
  - Do the same tests
- Open ConvFit
  - Load Sample `irs26176_graphite002_red.nxs` (from systemtest data)
  - Load Resolution `irs26173_graphite002_res.nxs` (from systemtest data)
  - Select Fit Type: Two Lorentzians
  - Do the same tests
